docs/configuration.md

```diff
@@ -1,339 +1,278 @@
-  the other build types (See _Samples_ section below).
-- `release` build type does not appear as a misc qualifier, as do other build
-  types, in the build artefact names when running `east release` command (see
-  _Release command_ section below).
-- Released firmware that has a `release` build type is considered as one and
-  only firmware that should be used in the production.
-
-## Applications
-
-The application is the main program in an `east` workspace. The most common the
-use-case is a GitHub repo with one application, however, repos with multiple
-applications are also supported.
-
-Depending on the number of applications the structure of the project needs to be
-adjusted:
-
-- If there is only one application, then its folders and files
-  (`CmakeLists.txt`, `conf`, `src`, etc.) can be directly placed into the `app`
-  folder.
-- If there are multiple applications, then their folders and files need to be
-  placed under separate folders inside the `app` folder, for example, below is a
-  structure for the `multi_app` project that has two applications `nrf52` and
-  `nrf91`:
+To understand how `east pack` works, it is important to know how to use Twister and how to specify
+build targets with `sample.yaml` files. Each target built by Twister is called a **test-suite** by
+the twister documentation, and will also be referred to as such in this document. For more details
+on how to use Twister to generate build artifacts, see our
+[Zephyr template repo](https://github.com/IRNAS/irnas-zephyr-template) and read the docs there.
 
-```
-example_project
-├── app
-│   ├── nrf52   # Here are CmakeLists.txt, conf, src, etc.
-│   ├── nrf91   # Here are CmakeLists.txt, conf, src, etc.
-├── boards
-├── CMakeLists.txt
-├── common
-├── drivers
-├── Kconfig
-├── samples
-├── west.yml
-└── zephyr
-```
+The `pack` key in `east.yml` specifies which files to copy for each of the test-suites. It has the
+following sub-keys:
 
-To configure an application in `east.yml`, the user must specify which
-`west-boards` and `build-types` are available.
+- `artifacts` - a list of files to copy for all test-suites.
+- `projects` - a list of specific test-suites with additional configuration. This allows you to
+  overwrite the default artifacts or specify additional artifacts for specific test-suites. Each
+  project in the list has the following:
+  - `name` - the name of the test-suite. This **must match** the name of the test-suite as defined
+    in the `sample.yaml` file.
+  - `artifacts` - a list of files to copy for this specific test-suite in addition to files in
+    `artifacts.pack`.
+  - `overwrite_artifacts` - a list of files to copy for this specific test-suite, overwriting the
+    default artifacts specified in `pack.artifacts`. When using this, do not use `artifacts`.
+- `extra` - a list of additional files to copy that are not part of the twister output, but are
+  created by other means.
```

Comment:
Maybe a possible improvement (we can talk about this): write all fields as full paths relative to the `pack.` fiels, so `artifacts` -> `project.artifacts`.